lib/model/special_triggers.dart

```diff
@@ -0,0 +1,28 @@
+import 'dart:math';
+
+import 'package:treasure_of_the_high_seas/model/resource.dart';
+
+import 'hand.dart';
+
+bool checkMutiny(Hand playerHand) {
+
+  // Mutiny next card if resource count over 15
+  return playerHand.cards.length > 15;
+
+}
```

Comment:
Dart lets you inline single line methods like this:

```
bool checkMutiny(Hand playerHand) => playerHand.cards.length > 15;
```

Reply:
Agreed, done.